services/brig/src/Brig/IO/Intra.hs

```diff
@@ -195,20 +195,27 @@ onPropertyEvent orig conn e =
       (pure $ list1 orig [])
 
 onClientEvent ::
+  ( MonadIO m,
+    Log.MonadLogger m,
+    MonadReader Env m,
+    MonadMask m,
+    MonadHttp m,
+    HasRequestId m
+  ) =>
   -- | Originator of the event.
   UserId ->
   -- | Client connection ID.
   Maybe ConnId ->
   -- | The event.
   ClientEvent ->
-  (AppT r) ()
+  m ()
```

Comment:
Just as a heads up, changes like these make negative progress towards the polysemization effort.

Reply:
Ah, sorry, it was an oversight. I'll try not to introduce any mtl style code anymore

Reply:
Thanks! Greatly appreciated :)